test/lib/apq_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule ApqTest do
+  use ExUnit.Case
+
+  test "README install version check" do
```

Comment:
Unrelated, but this simple test ensures that the version specified in the README is up-to-date with the version in `mix.exs`